app/views/pages/incompany/incompany.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>InCompany</h1>
```

Comment:
why is this file called  `pages/incompany/incompany.html.erb` instead of `pages/incompany.html.erb`?